install

```diff
@@ -202,9 +203,14 @@ main() {
     done
 }
 
-main "$@"
 
+PLATFORM=$(uname)
+if [ PLATFORM != "Linux" ]; then
```

Comment:
- `if [ PLATFORM != "Linux" ]`  should be: `if [ $PLATFORM != "Linux" ]`
- since we are on bash you can use `[[` instead of `[`

I totally agree with the platform check.

What about:
```
if [[ "${OSTYPE/-*}" != "linux" ]] ; then
...

```